src/pages/HomePage/HomePage.js

```diff
@@ -8,18 +8,37 @@ import Rating from './components/Rating/Rating'
 import Prices from './components/Prices/Prices'
 import Products from './components/Products/Products';
 import { Button } from 'react-bootstrap'
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { clearAllFilter } from '../../store/slices/ProductSlice'
 
 function HomePage() {
+
+    const dispatch = useDispatch()
+    const [ isClearFilter, setIsClearFilter ] = useState(false)
+    const filter = useSelector( state => state.products.filter)
+
+    useEffect(() => {
+        const filterValuesArr = [filter.name_like, filter.brand_like, filter.categories_like, filter.rating_like, filter._sort, filter._order, filter.type_like]
```

Comment:
```suggestion
        const filterValuesArr = Object.values(filter)
```
Thử như này xem nhé
Ngoài ra còn có Object.keys và Object.entries

Reply:
Chỗ này chưa thấy sửa